src/pages/api/models/column.ts

```diff
@@ -0,0 +1,22 @@
+export interface Column {
+  id: string;
+  name: string;
+  position: number;
+  color: string;
+}
+
+export const ColumnSchema = {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.